docs/banzai_nres/index.rst

```diff
@@ -8,3 +8,135 @@ Reference/API
+* | 'PRECISN' This should be between 4 and 10 m/s . High m/s values above 20 might indicate a failure of the wavelength
+  | solution, or extremely low S/N on the frame.
+* | 'CHISQ' between 0.2 and 5. An extremely high number here might mean errors are massively underestimated and
+  | vice-versa.
+* | 'LINENUM' between 3500 and 5000 (i.e. 1750 and 2500 lines per fiber). The code as is limits this value to 2500
+  | features per fiber.
+* | 'MLINENUM' should be roughly 50-80% of LINENUM, usually 70%. E.g. 3500 for 5000 total lines, or 2450 for 3500 total
+  | lines. If 'MLINENUM' is below 1000, your wavelength solution is probably overconstrained in the center of the
+  | detector and the edges are missed. This frame is probably too low of signal to noise.
+
+
+If all the above checks pass, you most likely have a very good wavelength solution. If any fail, you may want to dig
+into the frame deeper and look at an extracted science spectrum that uses this calibration. The end all test is to look
+at the wavelength overlap regions. You want the extracted spectral lines between two orders to agree almost exactly in
+the common wavelength overlap regions between those two orders. If that is not true, the wavelength calibration failed
+without a doubt.
+
+
+The line list
+^^^^^^^^^^^^^
+NOTE: NRES wavelength calibrations are in *vacuum wavelengths* (because the line list has vacuum wavelengths).
+
+Included in banzai_nres/data is the ThAr_atlas_ESO_original_air.txt which was fetched from
+http://www.eso.org/sci/facilities/paranal/instruments/uves/tools/tharatlas.html on August 27 2020. The line list
+that we use is banzai_nres/data/ThAr_atlas_ESO_vacuum.txt, which is the same list but converted to vacuum wavelengths.
+We generated ThAr_atlas_ESO_vacuum.txt as follows:
+
+We converted every line from ThAr_atlas_ESO_original_air.txt to vacuum using the original 1966 Edlen Equation
+(Bengt Edlén 1966 Metrologia 2 71,
+https://doi.org/10.1088/0026-1394/2/2/002) for the index of refraction of air.
+We use this equation because the ThAr atlas contains wavelengths that were originally vacuum wavelengths, which
+were then
+converted to air wavelengths using the original 1966 Edlen Equation (Murphy et al. 2007,
+DOI 10.1111/j.1365-2966.2007.11768.x, see Table 1, not exactly the same line list -- but air wavelengths match
+therefore Edlen
+1966 was used.). See the note below for more details.
+
+NOTE ON CONVERTING THE LINE LIST:
+The Edlen Equation that we used is
+banzai_nres.utils.wavelength_utils.index_of_refraction_Edlen . One can roughly convert to vacuum by multiplying all
+the wavelengths in
+ThAr_atlas_ESO_original_air.txt by the index of refraction at each line's AIR wavelength. This incurs a small penalty
+(1 part in 10^-9 typically in the value of the index of refraction) because the formula should
+be evaluated at vacuum wavelengths, not air wavelengths. For context, the original Edlen equation differs by
```

Comment:
The end of this feels like may be more detailed than the user needs to know. We may want to streamline it a bit.

Reply:
This is mostly for us so that we know how we arrived at the line list we use.